lib/logzio-nodejs.js

```diff
@@ -113,6 +114,11 @@ LogzioLogger.prototype.log = function(msg) {
     msg = _.assign(msg, this.extraFields);
     msg.type = this.type;
 
+    if (this.addNanoSecs) {
+        var time = process.hrtime();
+        msg['@timestamp_nano_secs'] = time[1];
```

Comment:
Yes, you're right, so if you want to order all the logs by time, it won't help.
If we concatenate the two fields like this :
`msg['@timestamp_nano_secs'] = parseInt(time[0] + time[1])`
Would that be ok ?